configs/base.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
As part of this PR we should include updates to the README.md explaining how to include this tsconfig in a project.